Allow to build against qt-6.5.

Re-enable the "OLD" behaviour for finding boost until our boost
package contains BoostConfig.cmake.

Index: CMakeLists.txt
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -215,7 +215,7 @@ find_soft_mandatory_package(ENABLE_QT5 Qt5Xml ${QT5_VE
 find_soft_mandatory_package(ENABLE_QT5 Qt5Widgets ${QT5_VERSION})
 find_soft_mandatory_package(ENABLE_QT5 Qt5Test ${QT5_VERSION})
 
-set(QT6_VERSION "6.2")
+set(QT6_VERSION "6")
 SET(QT_NO_CREATE_VERSIONLESS_TARGETS ON)
 find_soft_mandatory_package(ENABLE_QT6 Qt6Core ${QT6_VERSION})
 find_soft_mandatory_package(ENABLE_QT6 Qt6Gui ${QT6_VERSION})
@@ -225,7 +225,7 @@ find_soft_mandatory_package(ENABLE_QT6 Qt6Test ${QT6_V
 # Check for Cairo rendering backend
 macro_optional_find_package(Cairo ${CAIRO_VERSION})
 
-find_package(Boost 1.74.0 CONFIG)
+find_package(Boost 1.74.0)
 if(Boost_FOUND)
   set(USE_BOOST_HEADERS ON)
 elseif(ENABLE_BOOST)
